smithy-base-plugin/src/main/java/software/amazon/smithy/gradle/tasks/BaseSmithyTask.java

```diff
@@ -34,33 +33,45 @@ abstract class BaseSmithyTask extends DefaultTask {
     BaseSmithyTask() {
         getFork().convention(false);
         getShowStackTrace().convention(ShowStacktrace.INTERNAL_EXCEPTIONS);
-        getResolvedCliClasspath().convention(getProject().getConfigurations().getByName("smithyCli"));
-        getBuildClasspath().convention(getProject().getConfigurations().getByName("smithyBuild"));
-        getRuntimeClasspath().convention(getProject().getConfigurations().getByName("runtimeClasspath"));
+
+        // By default, there are no build dependencies or models discovery
```

Comment:
This says "no", which I read as empty, but passes in a value. Can you clarify this? Also `models` should be `model`.